docs/use-github-in-bcgov/github-transition-guide.md

```diff
@@ -76,12 +76,14 @@ GitHub doesn't offer users a screen to confirm authentication. There are 2 alter
 
 2) Another way to validate linking: 
 
-> 1. Go to your profile by clicking your avatar on the top right
-> 2. Click "Settings"
-> 3. Click "Enterprises"
-> 4. Check if you're a member of "Government of British Columbia
-> 5. If you are a member, then you've successfuly authenticated
+> 1. (**Mandatory step**) Go to https://github.com/bcgov/ 
+> 2. Got to your profile by clicking your avatar on the top right
+> 3. Click "Settings"
+> 4. Click "[Sessions](https://github.com/settings/sessions)"
+> 5. Click "See more" button for the current web session
+> 6. If you're authenticated, the following SSO details will be displayed:
```

Comment:
this isn't accessible, it should describe which words they expect to see because a person who uses a screen reader won't be able to read the screenshot

Reply:
@jenreiher-bcgov Hi, just need some clarification, do you meant the alt text on the screenshot needs to be built out, or the direct links? I'm assuming you meant the screenshot though.

Reply:
You could improve it by improving the alt text, or by improving the sentence before - either one (or maybe there's a 3rd way I'm not seeing)! but as is it's inaccessible.

Reply:
@jenreiher-bcgov could you double check the alt text? I'll edit the previous sentence too.